.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+# Reference on this file: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+name: Test
+on:
+  push:
+      branches:
+        - master
+        - v3 # todo: Remove when v3 is merged into master
+  pull_request:
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+      - name: Install Node packages
+        run: yarn install
+      - name: Build
+        run: yarn build
+      - name: Test on BrowserStack
+        env:
+          # This is in plaintext on purpose. It has no privileged access to anything (this is a free account)
+          # and it allows us to run browserstack tests against PRs.
+          BROWSERSTACK_USERNAME: github13kRcPC
```

Comment:
I see your comment but still why not use Github secret here, it's doesn't look very hard.
Maybe someone exceeds our free quota on BrowserStack just because they're lazy to register their own account.

But you have to remove this commit from history in order to fix it now...

Reply:
The main reason is to be able to test external PRs automatically; GitHub secrets aren't available in PRs. We can change the token any moment. I had this concert too, I've make a quick research and have found out that other projects keep the keys open too.